src/gasman.c

```diff
@@ -1815,10 +1815,11 @@ static NOINLINE void GenStackFuncBags(void)
         }
     }
 
-    /* mark from registers, dirty dirty hack                               */
-    for ( p = (Bag*)((void*)RegsBags);
-          p < (Bag*)((void*)RegsBags)+sizeof(RegsBags)/sizeof(Bag);
-          p++ )
+    // mark from registers, dirty dirty hack: we treat the jmp_buf as a
+    // sequence of Bag values. Note that sizeof(jmp_buf) need not be a
+    // multiple of sizeof(Bag), hence the end condition looks slightly.
+    // unusual.
+    for (p = (Bag *)RegsBags; p + 1 <= (Bag *)(RegsBags + 1); p++)
```

Comment:
Is this related to backporting GitHub actions? I see that this is how the code already looks in the `master` branch, I don't object to making the change to `stable-4.11`, I'm just curious why it's here.

Reply:
I addressed this in the commit message (but forgot to update the PR description):
> The tweak in src/gasman.c resolves a compiler warning that'd cause a CI failure (as we compile with -Werror)

Reply:
Thanks for the explanation. 🙂 I normally look at the commit messages, but I forgot this time.